Brocfile.js

```diff
@@ -33,12 +38,11 @@ tree = replace( app.toTree(), {
         {
             match: /REPLACE_META_DESCRIPTION/g,
             replacement: require('./package.json')['description']
```

Comment:
Do you want to reference these as a shared property as have done with sl-ember-components?